### PR TITLE
[LIBS-321] Fix Buildkite Error Handling

### PR DESCRIPTION
**Pull request by Thris3n**

### CODE REVIEW

File: tests/deploy-websocket.bats
Comment (smilin-desperado) [inline tests/deploy-websocket.bats:1]: 🚀

File: bin/deploy-websocket [hunk 1]
```diff
@@ -33,8 +33,10 @@ assert_var description "Argument 3: DESCRIPTION is not set"
 log "Creating deployment on API Websocket Gateway"
 
 response="$(aws apigatewayv2 create-deployment --stage-name "$matrix" --description "$description-$matrix" --api-id "$api_gateway_ws_id")"
-failed_count="$(jq '.FailedEntryCount' <<<"$response")"
-if [ ! "$failed_count" = 0 ]; then
+deployment_status=$(jq -r '.DeploymentStatus' <<<"$response" | sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//')
```
Comment (smilin-desperado) [inline bin/deploy-websocket:36]: What does piping to sed do here?

**Review by smilin-desperado — APPROVED** [review]
LGTM. Left a question

File: bin/deploy-websocket (hunk 1 above)
Comment (Thris3n) [reply bin/deploy-websocket:36]: sed to remove any trailing whitespace characters from the value obtained from jq

**Pull request merged**